news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: osu!monthly #1
+date: 2022-07-28
+author: Soetch, Blushing, wowcake, 0x84f
+---
+
+This is the new community's news post : osu!monthly (a new one, again)!
+
+![](/wiki/shared/news/2022-08-01-osumonthly-1/banner-july.png)
+
+## Word of the editors
+
+It's been years since **osu!weekly's** release has been paused, 6 years since the community didn't have a regular post. Today, we're coming in with **__osu!monthly__**, a bigger format.
+
+We think we did a good job gathering information for **July**, but we started the 07/17, so we had only two week to write, review and commit this article. 
+
+It'll never be the same as osu!weekly, but we're working hard to propose a frequent format, please be kind with us, new writers! We're highly receptive to it, so feedback is highly appreciated!
+
+➥ *[Soetch](https://osu.ppy.sh/users/15970028), [Blushing](https://osu.ppy.sh/users/5927823), [wowcake](https://osu.ppy.sh/users/16121851) & [0x84f](https://osu.ppy.sh/users/7944724)*
+
+---
+
+### Navigation
+- [What's up on osu! ?](#whats-up-on-osu)
+- [Top Scores of the Month](#top-scores-of-the-month)
+- [New features on osu!lazer (& bonus article!)](#new-features-on-osulazer)
+- [Summary of this article](#summary-of-this-article)
+- [End Word](#end-word)
+
+---
+
+## What's up on osu! ? 
```

Comment:
I know this was used a few times in the article, but I'm not sure about placing question marks with a space in front of them. On the wiki, there are plenty example of `osu!?`.